spec/javascripts/health-care/components/ProgressButton.spec.jsx

```diff
@@ -17,33 +17,44 @@ describe('<ContinueButton>', () => {
 
     // TODO: figure out why this is failing (same issue as in SSN spec)
     xit('onButtonClick is required', () => {
-      SkinDeep.shallowRender(<ContinueButton/>);
-      sinon.assert.calledWithMatch(consoleStub, /Required prop `onButtonClick` was not specified in `ContinueButton`/);
+      SkinDeep.shallowRender(<ProgressButton/>);
+      sinon.assert.calledWithMatch(consoleStub, /Required prop `onButtonClick` was not specified in `ProgressButton`/);
     });
 
     it('onButtonClick must be a function', () => {
       SkinDeep.shallowRender(
-        <ContinueButton onButtonClick/>);
-      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `onButtonClick` of type `boolean` supplied to `ContinueButton`, expected `function`/);
+        <ProgressButton onButtonClick/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `onButtonClick` of type `boolean` supplied to `ProgressButton`, expected `function`/);
+    });
+
+    it('buttonText must be a string', () => {
+      SkinDeep.shallowRender(<ProgressButton buttonText/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `buttonText` of type `boolean` supplied to `ProgressButton`, expected `string`/);
+    });
+
+    it('buttonClass must be a string', () => {
+      SkinDeep.shallowRender(
+        <ProgressButton buttonClass/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `buttonClass` of type `boolean` supplied to `ProgressButton`, expected `string`/);
     });
   });
 
   it('has sane looking features', () => {
-    const tree = SkinDeep.shallowRender(<ContinueButton/>);
+    const tree = SkinDeep.shallowRender(<ProgressButton/>);
     const buttons = tree.everySubTree('button');
     expect(buttons).to.have.lengthOf(1);
   });
 
   it('calls handleContinue() on click', () => {
```

Comment:
I wonder if we want to change this test assertion to be `calls handle function on click`, since it could be any of 3 particular handle functions now.